### DISCUSSION TITLE
Request: Object visibility distance change?

### DISCUSSION THREAD

**Discussion by 40P3** [34736384/genshin-fps-unlock#377] [Ideas]
Dear project owners (I can never figure it out on Git who actually owns this, sorry I'm a noob),
Is there a way to unlock the visibility range of objects? Since day one I hate this terrible visibility distance, where objects, enemies are popping in only in a very close range. However our hardware can certainly do much better since years, all other games got pretty crazy visibility distance now.

I would love to get like double the current visibility distance or maybe 3x? I would be probably very happy with it. Although if that could be a choice with a distance slider that would be even more crazy good.

What you think is this doable?
Regards,
H0PE

**Comment by 40P3** [comment]
So no response then. :(

**Comment by 34736384** [comment]
Yes and no, but it requires code injection, which is not necessarily safe.
